src/app/submission/objects/submission-objects.effects.ts

```diff
@@ -232,18 +242,45 @@ export class SubmissionObjectEffects {
         catchError(() => observableOf(new DiscardSubmissionErrorAction(action.payload.submissionId))));
     }));
 
+  @Effect() addAllMetadataToSectionData = this.actions$.pipe(
```

Comment:
Please add a comment/TypeDocs to describe this new Effect.

Reply:
Added TypeDoc